_includes/scenarios/integrate/create-attribute-for-yourself.md

```diff
@@ -0,0 +1,99 @@
+
+From a technical point of view, the creation of a RelationshipAttribute corresponds to the creation of one [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) for yourself and one LocalAttribute for your peer, whereby their `content` is given by the RelationshipAttribute that is intended to be created and the `shareInfo` of both LocalAttributes contains a correspondingly suitable [LocalAttributeShareInfo]({% link _docs_integrate/data-model-overview.md %}#localattributeshareinfo).
+{: .notice--info}
+
+### Utilization of a ReadAttributeRequestItem
+
+You can use a [ReadAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#readattributerequestitem) to create a RelationshipAttribute in the context of a Relationship between you and your peer if you want the RelationshipAttributeValue to be set by your peer. Even if it seems misleading to use a ReadAttributeRequestItem to create a RelationshipAttribute, this terminology makes sense insofar as the RelationshipAttributeValue should be read from the peer in order to be able to create the RelationshipAttribute. All details on how to create a RelationshipAttribute using a ReadAttributeRequestItem can be found in the [Read Attribute from peer]({% link _docs_integrate/read-attribute-from-peer.md %}) guide.
+
+### Utilization of a CreateAttributeRequestItem
+
+You can use a [CreateAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#createattributerequestitem) to create a RelationshipAttribute in the context of a Relationship between you and your peer if you want the RelationshipAttributeValue to be set by yourself. Your peer can only accept or reject the creation of the RelationshipAttribute, but cannot modify the RelationshipAttributeValue. For full details on how to create a RelationshipAttribute using a CreateAttributeRequestItem, please refer to the [Create Attribute for peer]({% link _docs_integrate/create-attribute-for-peer.md %}) guide.
+
+### Utilization of a ProposeAttributeRequestItem
+
+You can use a [ProposeAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#proposeattributerequestitem) to create a RelationshipAttribute in the context of a Relationship between you and your peer if you want to propose a potentially suitable RelationshipAttributeValue to your peer, but your peer has the option to modify it before the RelationshipAttribute is created. A RelationshipAttribute that you want to create using a ProposeAttributeRequestItem must be owned by your peer. All details on how to create a RelationshipAttribute using a ProposeAttributeRequestItem can be found in the [Propose Attribute to peer]({% link _docs_integrate/propose-attribute-to-peer.md %}) guide.
```

Comment:
It's an interesting fact, that here only the peer is allowed as owner and that it's the only case that has this restriction. Maybe state explicitly in the other cases, that both you and the peer can be the owner.

Reply:
I have also added the information about the owner to the CreateAttributeRequestItem 👍  

With the ReadAttributeRequestItem and ShareAttributeRequestItem this is not easily done in one sentence, as in these cases a third party is also possible as the owner and I'd not like to hide this fact when I start talking about the owner. Preference for leaving out the owner part or going into more detail (or no strong opinion)?

Reply:
If it doesn't get too cluttered, I'd state the allowance of a ThirdParty owner here then, too.